kornia/geometry/transform/affwarp.py

```diff
@@ -222,6 +228,8 @@ def rotate(tensor: torch.Tensor, angle: torch.Tensor,
     if center is None:
         center = _compute_tensor_center(tensor)
 
+    align_corners = _parse_align_corners(align_corners, Resample.get(mode), old_default=False)
```

Comment:
Just to confirm that if the resample is hardcoded on purpose here and after, right?

Can I ask why not make it as a param?

Reply:
Not sure what you mean:

- **Why am I using `Resample` at all?** Because in #633 it was used to deprecate the plain `str` for the interpolation mode. I assumed it will be used everywhere in `kornia` and this just isn't implemented yet.
- **Why am I using `Resample` here?** If the above holds true, `Resample` should be the base `_parse_align_corners` should be working with. Since it is not yet used here for `mode` (and changing it would be out of scope of this PR), I only pass the enum to `_parse_align_corners` and leave everything else as is.

Reply:
See https://github.com/kornia/kornia/pull/806#discussion_r545600628